entry.go

```diff
@@ -223,6 +254,53 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 	return nil
 }
 
+func (te *TunaEntry) nanoPayClaim(stream *smux.Stream, npc *nkn.NanoPayClaimer, reverseBytesIn, reverseBytesOut []uint64, lastComputed, lastClaimed common.Fixed64, lastUpdate time.Time) {
```

Comment:
lastComputed, lastClaimed, lastUpdate should be pointer

Reply:
Argument name should not contain reverse as it's used for both forward and reverse mode